### PR TITLE
Tests for worker and work executor

### PR DESCRIPTION
**Pull request by mario-renau-alstom**
Added sample test for Worker and WorkExecutor

### CODE REVIEW

File: pekko-sample-distributed-workers-scala/src/test/scala/worker/WorkerSpec.scala [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      val deliveryProbe = createTestProbe[ConsumerController.Confirmed]()
+      val deliveredMessage = Worker.DeliveredMessage(deliveryProbe.ref, work, 1)
+
+      worker ! deliveredMessage
+      worker ! Worker.WorkComplete("Successful result")
+
+      deliveryProbe.expectMessageType[ConsumerController.Confirmed]
+    }
+  }
+}
```
Comment (pjfanning) [inline pekko-sample-distributed-workers-scala/src/test/scala/worker/WorkerSpec.scala:39]: please add new lines at end of files

File: pekko-sample-distributed-workers-scala/src/test/scala/worker/WorkerSpec.scala [hunk 2]
```diff
@@ -0,0 +1,39 @@
+package worker
+
+import org.apache.pekko
+import org.apache.pekko.actor.typed.delivery.ConsumerController
```
Comment (pjfanning) [inline pekko-sample-distributed-workers-scala/src/test/scala/worker/WorkerSpec.scala:4]: can you remove the org.apache from this import and the next one

File: pekko-sample-distributed-workers-scala/src/test/scala/worker/WorkerSpec.scala (hunk 1 above)
Comment (mario-renau-alstom) [reply pekko-sample-distributed-workers-scala/src/test/scala/worker/WorkerSpec.scala]: Fixed

File: pekko-sample-distributed-workers-scala/src/test/scala/worker/WorkerSpec.scala (hunk 2 above)
Comment (mario-renau-alstom) [reply pekko-sample-distributed-workers-scala/src/test/scala/worker/WorkerSpec.scala:3]: fixed

**Comment by pjfanning** [comment]
The CI builds failed because this file is not correctly formatted.

pekko-sample-distributed-workers-scala/src/test/scala/worker/WorkerSpec.scala

Could you run `sbt scalaftmtAll` ?

**Pull request closed without merging**